server/tests/src/test/java/org/infinispan/server/functional/XSiteHotRodCacheOperations.java

```diff
@@ -1,54 +1,101 @@
-               "        <backup site=\"NYC\" strategy=\"ASYNC\"/>" +
-               "     </backups>" +
-               "  </distributed-cache-configuration>" +
-               "</cache-container></infinispan>";
-   @Rule
-   public InfinispanServerTestMethodRule LON_SERVER_TEST = new InfinispanServerTestMethodRule(XSiteIT.LON_SERVERS);
+   public static final InfinispanXSiteServerRule SERVERS = XSiteIT.SERVERS;
 
    @Rule
-   public InfinispanServerTestMethodRule NYC_SERVER_TEST = new InfinispanServerTestMethodRule(XSiteIT.NYC_SERVERS);
+   public InfinispanXSiteServerTestMethodRule SERVER_TEST = new InfinispanXSiteServerTestMethodRule(SERVERS);
 
    @Test
    public void testHotRodOperations() {
-      String lonXML = String.format(LON_CACHE_XML_CONFIG, LON_SERVER_TEST.getMethodName());
+      String lonXML = String.format(XSiteIT.LON_CACHE_XML_CONFIG, SERVER_TEST.getMethodName());
+      RemoteCache<String, String> lonCache = SERVER_TEST.hotrod(LON)
+            .withServerConfiguration(new XMLStringConfiguration(lonXML)).create();
+      RemoteCache<String, String> nycCache = SERVER_TEST.hotrod(NYC).create(); //nyc cache don't backup to lon
 
-      RemoteCache<String, String> lonCache = LON_SERVER_TEST.hotrod()
+      insertAndVerifyEntries(lonCache, nycCache, false);
+   }
+
+   @Test
+   public void testHotRodOperationsWithDifferentCacheName() {
+      RemoteCache<String, String> lonCache = SERVER_TEST.hotrod(LON)
+            .createRemoteCacheManager()
+            .administration()
+            .createCache("lon-cache", new XMLStringConfiguration(LON_CACHE_CUSTOM_NAME_XML_CONFIG));
+
+      RemoteCache<String, String> nycCache = SERVER_TEST.hotrod(NYC)
+            .createRemoteCacheManager()
+            .administration()
+            .createCache("nyc-cache", new XMLStringConfiguration(NYC_CACHE_CUSTOM_NAME_XML_CONFIG));
+
+      insertAndVerifyEntries(lonCache, nycCache, true);
+   }
+
+   @Test
+   public void testHotRodOperationsWithOffHeapSingleFileStore() {
+      String lonXML = String.format(LON_CACHE_OFF_HEAP, SERVER_TEST.getMethodName());
+      RemoteCache<Integer, Integer> lonCache = SERVER_TEST.hotrod(LON)
             .withServerConfiguration(new XMLStringConfiguration(lonXML)).create();
-      RemoteCache<String, String> nycCache = NYC_SERVER_TEST.hotrod().create(); //must have the same name as LON cache
+      RemoteCache<Integer, Integer> nycCache = SERVER_TEST.hotrod(NYC).create(); //nyc cache don't backup to lon
 
-      lonCache.put("k1", "v1");
-      nycCache.put("k2", "v2"); //nyc cache don't backup to lon
+      //Just to make sure that the file store is empty
+      assertEquals(0, getTotaFileStoreEntries(lonXML));
+
+      IntStream.range(0, 300).forEach(i -> lonCache.put(i, i));
+
+      assertEquals(300, nycCache.size());
+      //Just to verify that file store is properly working
+      assertEquals(100, getTotaFileStoreEntries(lonXML));
+   }
+
+   private int getTotaFileStoreEntries(String lonXML) {
```

Comment:
typo: Tota => Total
also, it checks the number of entries in memory and not in the filestore.